packages/neuron-ui/src/components/NervosDAORecord/index.tsx

```diff
@@ -91,7 +91,7 @@ export const DAORecord = ({
     epochParser(withdrawEpoch || currentEpoch)
   )
 
-  const leftEpochs = Math.max(compensationEndEpochValue - currentEpochValue, 0)
+  const leftEpochs = Math.max((compensationEndEpochValue || 0) - currentEpochValue, 0)
```

Comment:
My mistake. When depositEpoch is `''`,  compensationEndEpochValue becomes `NaN` and should not be converted to 0 for use. 
If the `depositEpoch` in depositEpochList is not retrieved in time from the RPC request, the message should be hidden to avoid displaying incorrect information.

https://github.com/nervosnetwork/neuron/blob/e9eab5ee3851106181675e4adc4a19b2b1fad044/packages/neuron-ui/src/components/NervosDAO/hooks.ts#L358